interfaces/builtin/bluetooth_control.go

```diff
@@ -49,7 +49,10 @@ const bluetoothControlConnectedPlugAppArmor = `
   /sys/devices/**/bluetooth/**    rw,
 
   # Requires CONFIG_BT_VHCI to be loaded
+  # some mtk drivers (mt7668) expose stpbt, stpbtfwlog device node
```

Comment:
I'd put this in a separate section like so:
```
  # Requires CONFIG_BT_VHCI to be loaded
  /dev/vhci                       rw,

  # Device nodes for MTK combo driver
  /dev/stpbt                      rw,
  /dev/stpbtfwlog                 rw,
```

Reply:
I could only find this source code of the driver: https://github.com/MotorolaMobilityLLC/kernel-mtk/blob/f008e679440bbc0f77229df9d6520b2831bee8f8/drivers/misc/mediatek/connectivity/common/common_main/linux/stp_chrdev_bt.c

It seems like it's not really adding anything that would match with `SUBSYSTEM=="bluettooth"` rule. Have you checked that the access is not blocked by device cgroup?